data/deep missions.txt

```diff
@@ -356,11 +351,13 @@ event "deep: warlord detected"
 
 
 mission "Deep: Mystery Cubes 1"
+	minor
 	landing
 	name `Travel to <planet>`
 	description `Travel to <destination> to learn more about the situation with the warlord Beelzebub.`
 	source
-		government Republic "Free Worlds" Neutral Syndicate
+		government Republic "Free Worlds" Neutral Syndicate Independent
+		near Sol 1 100
```

Comment:
What's the purpose of this?

Reply:
No messages reaching the player if they are in Pug space during FW conflict.

Reply:
Alright.